actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -54,7 +54,7 @@ def parameters
                  rescue EOFError
                    query_parameters.dup
                  end
-        params.merge!(path_parameters)
+        params.merge!(path_parameters_utf8_encoding)
```

Comment:
Sorry, I spoke too soon.
Seems like we don't force everything to UTF8 anymore. https://github.com/rails/rails/commit/2eb0a6630ac3ace528de04b308d0250ac613a318